client/web/src/enterprise/batches/BatchSpecNode.tsx

```diff
@@ -256,7 +255,7 @@ const BinaryBatchWorkspaceFile: React.FunctionComponent<BatchWorkspaceFileConten
                 outline={true}
                 variant="secondary"
                 size="sm"
-                to={downloadUrl}
+                to={file.url}
```

Comment:
```suggestion
                to={downloadUrl}
```
This should be the `downloadUrl`, I can't remember, but if you use the `file.url` variable directly, are you able to download the file?

The reason I added the `generateFileDownloadLink` function is because that endpoint requires authentication, and just making the request directly with the button wouldn't pass that in.

Reply:
@BolajiOlajide I kept this as a draft because I didnt get to test it yet, currently reading the docs to set up a file mount but wanted to request review early since i know this has to go out soon so thanks! I will switch it.

Reply:
Alright. No problem at all.